redis.go

```diff
@@ -313,6 +313,16 @@ func (c *baseClient) initConn(ctx context.Context, cn *pool.Conn) error {
 		return err
 	}
 
+	if !c.opt.DisableIndentity {
```

Comment:
I think it should be `DisableIdentity`(typo)

And I didn't understand what is the reason to run it not in the pipeline?

Reply:
https://github.com/redis/go-redis/issues/2911
Client setinfo needs to be "fire and forget" and when executing in pipline we check for errors in pipline.

Reply:
While it should be DisableIdentity, that would now be a breaking change for existing users. We'll have to get to that in V10.